packages/apps/desktop-viewer-test/src/frontend/components/App.tsx

```diff
@@ -3,9 +3,12 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import "@itwin/itwinui-layouts-css/styles.css";
```

Comment:
id recommend adding this at the desktop frontend root, which happens to be `frontend/index.tsx`